zowe_conformance/terms_and_conditions.md

```diff
@@ -1,4 +1,24 @@
-# Zowe Conformance Program Terms and Conditions
+# Zowe Comformance Program Terms and Conditions
```

Comment:
Minor nit, "Comformance" => "Conformance"

Reply:
Tks for finding this - should be corrected now